torch/csrc/jit/export.cpp

```diff
@@ -696,11 +690,15 @@ void ScriptModuleSerializer::convertModel(
                                           // using appropriate function call
   model_def->set_proto_version(torch::ProtoVersion::PROTO_VERSION_NEWEST);
 
-  convertModule(
-      module, "", writer_.archiveName(), model_def->mutable_main_module());
+  // Serialize all code info.
+  convertClass(module.type());
 
-
-  writePickleArchive("attributes.pkl", pickled_ivalues_);
+  // Then pickle the module
+  Pickler pickler(&tensor_table_);
+  pickler.start();
+  pickler.addIValue(module.module_object());
+  pickler.finish();
+  writer_.writeRecord("module.pkl", pickler.stack().data(), pickler.stack().size());
```

Comment:
Let's not use the work "module". Afterall this archieve doesn't need to encode modules. How about `code/` for the code, and `data.pkl` for the data?